### DISCUSSION TITLE
ART parameters are abnormal

### DISCUSSION THREAD

**Discussion by j0110** [LSPosed/LSPosed#2090] [Q&A]
Hi,
I was wondering if there is a fix to the "ART parameters are abnormal" warning in Momo else than modifying props that is considered as a dirty solution...
Thanks for your answers !

**Comment by BadIDMan** [comment]
Hi,
I would like to ask the same: could you please provide a fix for "ART parameters are abnormal" trace detected by Momo?
I have installed latest LSposed framework module 1.8.6 (6712) Zygisk and just after that "ART parameters are abnormal" is detected.
I tried also with previous version and the result is the same.

A dirty fix provided here: https://forum.xda-developers.com/t/magisk-general-support-discussion.3432382/post-85951781 is not applicable because it is for Riru variant. In Zygisk there is no system.prop in /data/adb/modules/zygisk_lsposed/ folder.

Running Samsung Android 13 rooted with Magisk Alpha (25206) (34) version.

[EDIT] With latest Magisk 26101 "ART parameters are abnormal" trace is not detected by Momo anymore.
